config/initializers/rack_attack.rb

```diff
@@ -95,7 +95,7 @@ def remote_ip
         bantime: Figaro.env.otp_delivery_blocklist_bantime.to_i.minutes
       ) do
         # The count for the phone_number is incremented if the return value is truthy
-        req.get? && req.path == '/otp/new'
+        (req.get? && req.path == '/otp/new') || (req.get? && req.path == '/otp/send')
       end
```

Comment:
This is related to my previous comments about `otp/send` and `otp/new` being duplicative. We only need one.